exercises/practice/all-your-base/AllYourBase.pm

```diff
@@ -5,7 +5,7 @@ use Exporter qw<import>;
 our @EXPORT_OK = qw<rebase>;
 
 sub rebase {
-  my ($digits) = @_;
+  my ($input) = @_;
```

Comment:
`input` is non-descript, while `digits` tells the story.  I don't think we should change the tone to the programming level, rather than leaving it at the "story of the problem being solved" level.

Reply:
This was actually a mistake on my part previously. `digits` is only a single key, where the input itself is actually a hash containing digits, input base, and output base.

I'm happy to accept a better suggestion, but `digits` is definitely wrong here.